don't have 238 yet, but this also came with no changes when bumping it.
238 has newline stripping fixes but this probably doesn't matter since we don't
have it anyway so it's already broken
--
diff --git a/meson.build b/meson.build
index b16f36c..7d35b96 100644
--- a/meson.build
+++ b/meson.build
@@ -69,7 +69,7 @@ gobject_introspection = dependency('gobject-introspection-1.0', required: get_op
 
 
 if os_backend == 'linux'
-  gudev_dep = dependency('gudev-1.0', version: '>= 238')
+  gudev_dep = dependency('gudev-1.0', version: '>= 237')
   idevice_dep = dependency('libimobiledevice-1.0',
                            version : '>= 0.9.7',
                            required : get_option('idevice'))
